go-controller/pkg/util/net_linux.go

```diff
@@ -15,6 +15,74 @@ import (
 	utilnet "k8s.io/utils/net"
 )
 
+type NetLinkLibOperations interface {
+	LinkByName(ifaceName string) (netlink.Link, error)
+	LinkSetup(link netlink.Link) error
+	AddrList(link netlink.Link, family int) ([]netlink.Addr, error)
+	AddrDel(link netlink.Link, addr *netlink.Addr) error
+	AddrAdd(link netlink.Link, addr *netlink.Addr) error
+	RouteList(link netlink.Link, family int) ([]netlink.Route, error)
+	RouteDel(route *netlink.Route) error
+	RouteAdd(route *netlink.Route) error
+	RouteListFiltered(family int, filter *netlink.Route, filterMask uint64) ([]netlink.Route, error)
+	NeighAdd(neigh *netlink.Neigh) error
+	NeighList(linkIndex, family int) ([]netlink.Neigh, error)
+	ConntrackDeleteFilter(table netlink.ConntrackTableType, family netlink.InetFamily, filter netlink.CustomConntrackFilter) (uint, error)
+}
+
+type defaultNetLinkLibOper struct {
+}
+
+var netLinkOperations NetLinkLibOperations = &defaultNetLinkLibOper{}
+
+func (inst defaultNetLinkLibOper) LinkByName(ifaceName string) (netlink.Link, error) {
```

Comment:
since you are not using `inst` inside the function, can you drop `inst` as in

`func (defaultNetLinkLibOper) LinkByName(ifaceName string) (netlink.Link, error) {`

Reply:
sure, can drop `inst` as suggested